flybywiresim/a32nx
# fix(model): ecam pushbuttons light colour

Another check on #1410

## Summary of Changes


The buttons in the ECAM Control Panel incorrectly used the same green illumination as the FCU buttons. These PR fixes the color and gives them the warm white they should have.


## Screenshots (if necessary)


![ECP](https://user-images.githubusercontent.com/929769/129446494-966643df-4d94-47cd-93f9-4c9a3caac1e7.png)

## References

![image](https://user-images.githubusercontent.com/929769/129446532-29cb9658-b253-401b-ab4a-fd81529da0d2.png)

Video reference: https://www.youtube.com/watch?v=rTlaA7JpV5Y




## Additional context



Discord username (if different from GitHub):

## Testing instructions


1. Start a new flight
2. Interact with the ECAM buttons, verify the color is accurate. The buttons, when illuminated, should light up in a warm white.
3. Verify the similar button in the FCU still have the original color
4. Ensure the ANN LT TEST works as expected


## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page
